asciidoc/courses/aura-dashboards/modules/1-getting-started/lessons/2-accessing-aura-dashboards/questions/2-empty-dashboards-canvas.adoc

```diff
@@ -1,5 +1,5 @@
 [.question]
-= Getting access to Aura Dashboards
+= Understanding the Empty Canvas
```

Comment:
Understanding the empty canvas isn't covered in the lesson content. 

tbh - I would be tempted to not have a question for this lesson, it is essential just setup.